osbs/build/spec.py

```diff
@@ -190,6 +190,7 @@ def set_params(self, git_uri=None, git_ref=None,
 class ProdSpec(CommonSpec):
     git_branch = BuildParam('git_branch', allow_none=True)
     trigger_imagestreamtag = BuildParam('trigger_imagestreamtag')
+    build_imagestream = BuildParam('build_imagestream')
```

Comment:
Please move the imagestream code under `CommonSpec` so that simple builds can use it too (we should probably remove them in the future but that's another story ...).